migtests/tests/oracle-tests/ora-trunc-tests/ora-date-trunc-test/trunc_date_schema_data.sql

```diff
@@ -12,13 +32,11 @@ DECLARE
     tz1_tz TIMESTAMP WITH TIME ZONE;
     tz2_ts TIMESTAMP WITH TIME ZONE;
 BEGIN
-    SELECT TRUNC(TO_DATE('02-MAR-15','DD-MON-YY'), 'mi') into d1_date FROM DUAL;
-    SELECT TRUNC(TO_DATE('02-MAR-16','DD-MON-YY'), 'ww') into d2_date FROM DUAL;
-    SELECT TRUNC(TO_DATE('02-MAR-17','DD-MON-YY'), 'iw') into d3_date FROM DUAL;
-    SELECT TRUNC(TO_TIMESTAMP('02-MAR-18','DD-MON-YY'), 'mm') into tz1_tz FROM DUAL;
-    SELECT TRUNC(TO_TIMESTAMP('02-MAR-19','DD-MON-YY'), 'YEAR') into tz2_ts FROM DUAL;
-    INSERT INTO TRUNC_TEST_1 (d1,d2,d3,tz1,tz2) VALUES(d1_date,d2_date,d3_date,tz1_tz,tz2_ts);
+    d1_date := TRUNC_DATE('02-MAR-15', 'mi');
+    d2_date := TRUNC_DATE('02-MAR-16', 'ww');
+    d3_date :=TRUNC_TIME_STAMP('02-MAR-18', 'mm');
+    tz1_tz :=TRUNC_TIME_STAMP('02-MAR-18', 'mm');
+    tz2_ts := TRUNC_TIME_STAMP('02-MAR-19', 'YEAR');
+    INSERT INTO TRUNC_TEST (d1,d2,d3,tz1,tz2) VALUES(d1_date,d2_date,d3_date,tz1_tz,tz2_ts);
 END;
 /
-
-CREATE OR REPLACE SYNONYM TEST_SYNONYM for TRUNC_TEST_1;
```

Comment:
Why is this removed? Was this added by you?

Reply:
Yes, I removed it as the test schema was changed by me. This test was for validated truncated date functionality & the presence of synonyms here was not a requirement.

Reply:
@chetank-yb `CREATE OR REPLACE SYNONYM TEST_SYNONYM for TRUNC_TEST_1;` I added this DDL for Synonym test. Let me know if you already have synonym test, as we made a fix for Synonym now they are exporting correctly without source_schema name and hence can imported in target db schema. Hence, can have test for it.

Reply:
I will add a new test for the synonym test @priyanshi-yb . The schema for this test is completely different now